lib/discount.js

```diff
@@ -0,0 +1,59 @@
+class Discount {
+	constructor(){
+	}
+	eval(dfr){
+		return undefined;
+	}//dfr = t, where d_t is the discount given to the action t time steps ahead
+}
+
+class GeometricDiscount {//extends Discount{ //Includes no discount with gamma = 1
+	constructor(gamma){
+		this.gamma = gamma
+	}
+	eval(dfr){
```

Comment:
Ideally we should make all the discount classes extend Function, so you can just call it:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions